.github/workflows/smoketests.yml

```diff
@@ -54,7 +54,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       fail-fast: false
-      max-parallel: 1
+#      max-parallel: 1
```

Comment:
If its unecessary lets delete it instead of commenting out

Reply:
sure, deleted